chrome/content/zotero/xpcom/integration.js

```diff
@@ -2038,6 +2046,9 @@ Zotero.Integration.Session.prototype.getCiteprocLists = function() {
  * Updates the list of citations to be serialized to the document
  */
 Zotero.Integration.Session.prototype._updateCitations = async function () {
+	if (Zotero.Prefs.get('cite.useCiteprocRs')) {
```

Comment:
Let's at least change this to `useCiteprocRS`.

Reply:
This doesn't really concern me, but it isn't an acronym, it's an abbreviation. JavaScript to JS is a quasi acronym, but Rust is one word that you never spell RuSt.

Reply:
OK, that's fair. We can go back to Rs.